Tests/Linq/Linq/NullableBoolTests.cs

```diff
@@ -0,0 +1,289 @@
+﻿using System;
+using System.Linq;
+
+using LinqToDB;
+
+using NUnit.Framework;
+
+namespace Tests.Linq
+{
+	[TestFixture]
+	public class NullableBoolTests : TestBase
+	{
+		record NullableBoolClass
+		{
+			public bool? Value;
+		}
+
+		const string ProvidersThatDoNotSupportNullableBool = $"{TestProvName.AllAccess},{TestProvName.AllSybase},{ProviderName.ClickHouseOctonica}";
```

Comment:
```suggestion

```suggestion

		// octonica blocked by https://github.com/Octonica/ClickHouseClient/issues/56
		const string ProvidersThatDoNotSupportNullableBool = $"{TestProvName.AllAccess},{TestProvName.AllSybase},{ProviderName.ClickHouseOctonica}";
```
				const string ProvidersThatDoNotSupportNullableBool = $"{TestProvName.AllAccess},{TestProvName.AllSybase},{ProviderName.ClickHouseOctonica}";
```